axum-core/src/response/into_response_parts.rs

```diff
@@ -57,7 +57,10 @@ use std::{
 /// }
 ///
 /// // We can now return `SetHeader` in responses
-/// async fn handler() -> impl IntoResponse {
+/// //
+/// // Note that returning `impl IntoResponse` might be easier if the response has many parts to
+/// // it. The return type is written out here for clarity.
+/// async fn handler() -> (SetHeader<'static>, SetHeader<'static>, &'static str) {
 ///     (
```

Comment:
@davidpdrsn doesn't the Impl IntoReponse interface recurse over the tuple essentially calling `.into_reponse()` on each tuple element?

Reply:
It does yes.